app/AvailablePlugin/IdentifierEnroller/Controller/IdentifierEnrollerCoPetitionsController.php

```diff
@@ -59,12 +59,37 @@ protected function execute_plugin_collectIdentifier($id, $onFinish) {
       $this->redirect($onFinish);
     }
 
+    // Get the CO Person ID
+    $coPersonId = $this->CoPetition->field('enrollee_co_person_id', array('CoPetition.id' => $id));
+
+    // Get the CoPerson Identifiers.
+    // If we already have a type of identifier that matches the one controlled by the plugin instance
+    // skip
+    $args = array();
+    $args['joins'][0]['table'] = 'co_people';
+    $args['joins'][0]['alias'] = 'CoPerson';
+    $args['joins'][0]['type'] = 'INNER';
+    $args['joins'][0]['conditions'][0] = 'CoPerson.id=Identifier.co_person_id';
```

Comment:
Care needs to be taken when joining tables since this can bypass ChangelogBehavior and return deleted rows.

But... what does this join do in the first place? Why not just find Identifiers where `Identifier.co_person_id = $coPersonId`?